### ISSUE TITLE
Issues with YouTube history

### ISSUE THREAD

**Issue by milindgoel15** [StevenBlack/hosts#1379]
Hi, from the last few months I figured some issue with YouTube history and several others users too faced. When we use ur host in adaway fully updated to latest, the yt history will not work. When we disable and use the rest host sources provided by adaway by default, it starts to work. 

Even whitelisting the *s.youtube.com doesn't work when we use ur host source.

Can you fix this issue if possible.

**Comment by tbalden** [comment]
yes, i've just bumped into this one as well. Thanks for looking into it!

**Comment by gghhkm** [comment]
Yeah, I was having this issue and followed the upstream to here.

**Comment by LittleControl** [comment]
Has anyone solved this problem？

**Comment by p1r473** [comment]
whitelist ```s.youtube.com```

**Comment by LittleControl** [comment]
> whitelist `s.youtube.com`

Solved it, thanks

**Comment by milindgoel15** [comment]
Whitelisting doesn't fix it. Only disabling stevenblack does

**Comment by Sego1234** [comment]
i removed the s.youtube.com domain from the host list
works fine

**Comment by theltalpha** [comment]
> Whitelisting doesn't fix it. Only disabling stevenblack does

Hallo! When you say "whitelisting", what do you exactly do? I simply removed (deleted) the s.youtube.com line, and theYoutube history then works.

**Comment by milindgoel15** [comment]
> > Whitelisting doesn't fix it. Only disabling stevenblack does
> 
> Hallo! When you say "whitelisting", what do you exactly do? I simply removed (deleted) the s.youtube.com line, and theYoutube history then works.

Ok it had been long time since I used host source of stevenblack. Earlier there was an issue where allowing "s.youtube.com" doesn't use to work even trying with *. And disabling the source(removing from active sources) helps.

So just now after u replied to this thread, I tried it and after whitelisting with active stevenblack source, it worked.(Not without whitelisting) 

So it's still somewhat an issue with this source but I am fine with what we are getting. If dev of this source can fix it so that we don't need to whitelist the url manually, that would be much appreciated. I am closing this issue for now.
